selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/news/NewsLocator.java

```diff
@@ -0,0 +1,20 @@
+package org.exoplatform.platform.qa.ui.selenium.locator.news;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.$;
+
+public class NewsLocator {
+
+  public static final SelenideElement NEWS_COMPOSER_TAB = $(byId("ActivityComposerExt")).find(byClassName("UINewsActivityComposer"));;
+
+  public static final By NEWS_COMPOSER_POST_BUTTON = By.id("newsPost");
```

Comment:
Why using the Selenium syntaxe instead Selenide with CSS selector syntax which is more readable ?
```suggestion
  public static final SelenideElement NEWS_COMPOSER_POST_BUTTON = $("#newsPost");
```